pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -224,25 +242,26 @@ in (buildEnv {
     # sort entries to improve reproducibility
     [[ -f "$TEXMFSYSCONFIG"/web2c/updmap.cfg ]] && sort -o "$TEXMFSYSCONFIG"/web2c/updmap.cfg "$TEXMFSYSCONFIG"/web2c/updmap.cfg
 
-    perl "$out"/share/texmf/scripts/texlive/mktexlsr.pl --sort "$out"/share/texmf-* # to make sure
+    perl "$TEXMFDIST"/scripts/texlive/mktexlsr.pl --sort "$TEXMFSYSCONFIG" "$TEXMFSYSVAR" # to make sure
```

Comment:
(Here?)

Reply:
Rather, it is quite likely that this call is not necessary. I left it because it was already there, and there may have been a good reason (the comment "to be sure" is scarce on details, unfortunately!).